### PR TITLE
`Account` examples

### PR DESCRIPTION
**Pull request by war-in**
Closes #

## Introduced changes

Examples for the `Account` API

##

- [ ] This PR contains breaking changes

### CODE REVIEW

File: starknet_py/tests/e2e/docs/code_examples/test_account.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        ),
+        max_fee=int(1e15),
+    )
+    # or
+    # docs-end: execute
+    call1 = call2 = Call(
+        to_addr=contract_address,
+        selector=get_selector_from_name("increase_balance"),
+        calldata=[123],
+    )
```
Comment (cptartur) [inline starknet_py/tests/e2e/docs/code_examples/test_account.py:57]: I'm not sure about `call1 = call2` being readable enough.

File: starknet_py/tests/e2e/docs/code_examples/test_account.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+@pytest.mark.asyncio
+@patch(
+    "starknet_py.net.account.account.Account.get_balance",
+)
+async def test_get_balance(gateway_account):
+    account = gateway_account
+    # docs-start: get_balance
+    eth_balance = await account.get_balance()
+    # or
```
Comment (cptartur) [inline starknet_py/tests/e2e/docs/code_examples/test_account.py:71]:
```suggestion
    # or with custom token contract address
```

File: starknet_py/tests/e2e/docs/code_examples/test_account.py (hunk 1 above)
Comment (war-in) [reply starknet_py/tests/e2e/docs/code_examples/test_account.py:57]: But it is not included in the docs 😅

Comment (cptartur) [reply starknet_py/tests/e2e/docs/code_examples/test_account.py:57]: Oh, right. Disregard then.

**Review by cptartur — APPROVED** [review]

File: starknet_py/tests/e2e/docs/code_examples/test_account.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        signer=StarkCurveSigner(
+            account_address=0x123,
+            key_pair=KeyPair(12, 34),
+            chain_id=StarknetChainId.TESTNET,
+        ),
+    )
+    # or
+    account = Account(
+        address=0x123,
+        client=FullNodeClient(node_url="your.node.url", net=TESTNET),
```
Comment (drknzz) [inline starknet_py/tests/e2e/docs/code_examples/test_account.py:32]: `net` could be set to "testnet" in one example

File: starknet_py/tests/e2e/docs/code_examples/test_account.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        chain=StarknetChainId.TESTNET,
+    )
+    # docs-end: init
+
+
+@pytest.mark.asyncio
+async def test_execute(gateway_account, contract_address):
+    account = gateway_account
+    # docs-start: execute
+    resp = await account.execute(
```
Comment (drknzz) [inline starknet_py/tests/e2e/docs/code_examples/test_account.py:43]: An example for `auto_estimate=True` maybe?

**Review by drknzz — APPROVED** [review]

**Pull request merged**